lstchain/calib/camera/calibrator.py

```diff
@@ -130,28 +127,27 @@ def _initialize_correction(self):
                     table = '/tel_' + str(telid) + '/pixel_status'
                     next(h5_table.read(table, self.mon_data.tel[telid].pixel_status))
         except:
-            self.log.error(f"Problem in reading calibration file {self.calibration_path}")
+            self.log.exception(f"Problem in reading calibration file {self.calibration_path}")
+            raise FileNotFoundError(f"Problem in reading calibration file {self.calibration_path}")
```

Comment:
why this duplication of errors?

Reply:
that is based on the comment here: https://github.com/cta-observatory/cta-lstchain/pull/331#discussion_r405313711